src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -508,20 +509,12 @@ public void registerChargeableBlock(boolean vanilla, int capacity, ItemHandler..
 		addItemHandler(handlers);
 		registerChargeableBlock(vanilla, capacity);
 	}
-
-	public EnergyTicker getEnergyTicker() {
-		return this.energy;
-	}
 	
 	public BlockMenu getBlockMenu(Block b) {
 		return BlockStorage.getInventory(b);
 	}
 	
 	public void addWikipage(String page) {
-		Slimefun.addWikiPage(this.getName(), "https://github.com/mrCookieSlime/Slimefun4/wiki/" + page);
-	}
-	
-	public boolean isAddonItem() {
-		return this.addon;
+		Slimefun.addWikiPage(this.getID(), "https://github.com/mrCookieSlime/Slimefun4/wiki/" + page);
```

Comment:
Not related to this PR, but it's TheBusyBiscuit now.

Reply:
Then, it's time to fix it :D